spec/ttml1.xml

```diff
@@ -8970,6 +8970,10 @@ attribute
 and (2) is capable of displaying or generating an output display
 signal that distinguishes between at least sixteen (16) values of color,
 including all primary and secondary colors of the SRGB color space.</p>
+<note role="explanation">
+<p>Support for this feature is intended to imply support for the following features: <code>#backgroundColor-block</code>,
```

Comment:
Change "is intended to imply" to "implies".

Reply:
@skynavga The form "intended to imply support" is used throughout feature definitions. Why do it differently here?

Reply:
In that case, I withdraw my comment.

Reply:
@skynavga Please approve the PR then.